gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/connection.js

```diff
@@ -95,11 +95,15 @@ class Connection extends EventEmitter {
     this.isOpen = false;
     this.traversalSource = options.traversalSource || 'g';
     this._authenticator = options.authenticator;
+    this._session = options.session || false;
```

Comment:
We should add `options.session` to the `Connection` and to the `Client` jsdoc.

Reply:
ok, thanks